platform/ios/src/MGLMapView.h

```diff
@@ -1941,9 +1941,9 @@ MGL_EXPORT
  */
 @property (nonatomic) MGLMapDebugMaskOptions debugMask;
 
-@property (nonatomic, getter=isDebugActive) BOOL debugActive __attribute__((unavailable("Use -debugMask and -setDebugMask:.")));
+@property (nonatomic, getter=isDebugActive) BOOL debugActive __attribute__((unavailable("Use `-debugMask` and `-setDebugMask:`.")));
 
-- (void)toggleDebug __attribute__((unavailable("Use -setDebugMask:.")));
+- (void)toggleDebug __attribute__((unavailable("Use `-setDebugMask:`.")));
```

Comment:
By the way, we can delete these members. They were marked unavailable quite some time ago, possibly since v3.0.

Reply:
Might as well!

Reply:
→ https://github.com/mapbox/mapbox-gl-native/pull/15000